### PR TITLE
Assorted broken link fixes

### PR DESCRIPTION
**Pull request by hamishwillee**
I keep finding new broken links, by finding better link checker tools. 

FYI me, 
- These were found using https://github.com/tcort/markdown-link-check
- BLC linkcheck (installed via `npm install broken-link-checker -g`) is OK but exclude globs don't appear to work (`blc https://docs.px4.io -rofi --exclude /v1.11 --exclude /v1.10 --exclude /v1.9 --exclude /master/zh --exclude /master/ko --filter-level 3 >blc_linkcheck.log`. Also missing many cases found by markdownlink check.)
- https://github.com/filiph/linkcheck  - installed via dart - works OK using `linkcheck https://docs.px4.io/master/en/ >linkcheck_linklog.txt -i` and `-e`

### CODE REVIEW

File: en/advanced_features/traffic_avoidance_adsb.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ PX4 traffic avoidance works with ADS-B or FLARM products that supply transponder
 
 It has been tested with the following devices:
 - [PingRX ADS-B Receiver](https://uavionix.com/product/pingrx/) (uAvionix)
-- [FLARM](https://flarm.com/products/powerflarm/uav/)
+- [FLARM](https://flarm.com/products/powerflarm/uav/)
```
Comment (hamishwillee) [inline en/advanced_features/traffic_avoidance_adsb.md:14]: Link broken. Any suggestions for replacement?

File: en/simulation/gazebo.md [hunk 2]
```diff
@@ -340,7 +340,7 @@ The camera also supports/responds to the following MAVLink commands: [MAV_CMD_RE
 :::
 
 :::note
-The simulated camera is implemented in [PX4/sitl_gazebo/src/gazebo_geotagged_images_plugin.cpp](https://github.com/PX4/sitl_gazebo/blob/master/src/gazebo_geotagged_images_plugin.cpp).
+The simulated camera is implemented in [PX4/sitl_gazebo/src/gazebo_geotagged_images_plugin.cpp](https://github.com/PX4/sitl_gazebo/blob/master/src/gazebo_geotagged_images_plugin.cpp).
```
Comment (hamishwillee) [inline en/simulation/gazebo.md:343]: File missing - suggestions: https://github.com/PX4/sitl_gazebo/blob/master/src/gazebo_geotagged_images_plugin.cpp

File: en/ros/offboard_control.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ Small low power examples:
 Larger high power examples:
 * [Intel NUC](http://www.intel.com/content/www/us/en/nuc/overview.html)
 * [Gigabyte Brix](http://www.gigabyte.com/products/list.aspx?s=47&ck=104)
-* [Nvidia Jetson TK1](https://developer.nvidia.com/jetson-tk1)
+* [Nvidia Jetson TK1](https://developer.nvidia.com/jetson-tk1)
```
Comment (hamishwillee) [inline en/ros/offboard_control.md:62]: @bkueng This appears to be discontinued. Do you know of a natural successor? Or who else I could ask?

**Comment by hamishwillee** [comment]
Merging. There are still some broken links identified in here, but fixing what is known good.

File: en/ros/offboard_control.md (hunk 3 above)
Comment (bkueng) [reply en/ros/offboard_control.md:62]: I'm not completely up to date, but typically the TX2 is often used.

File: en/simulation/gazebo.md (hunk 2 above)
Comment (bkueng) [reply en/simulation/gazebo.md:343]: https://github.com/PX4/PX4-SITL_gazebo/blob/master/src/gazebo_camera_manager_plugin.cpp

**Pull request merged**